cmd/config/set/da.go

```diff
@@ -26,6 +26,89 @@ func setDA(rlpCfg roller.RollappConfig, value string) error {
 	return updateDaConfig(rlpCfg, daValue)
 }
 
+func setDARPC(rlpCfg roller.RollappConfig, value string) error {
+	daCfgDirPath := filepath.Join(rlpCfg.Home, consts.ConfigDirName.DALightNode)
+	_, err := filesystem.DirNotEmpty(daCfgDirPath)
+	if err != nil {
+		return err
+	}
+
+	rlpCfg.DA.RpcUrl = value
+	switch rlpCfg.DA.Backend {
+	case consts.Celestia:
```

Comment:
base url here is always the locally running light client, the rpc command should not update the url for celestia